### PR TITLE
Improve PR template

### PR DESCRIPTION
**Pull request by dozortsev**
## ❓ Context  

Want people remind people to add mockups if they exist (which quote often they are), as it quote useful context info while code review.

But I'm not sure I added in most proper place, open for suggestions 🙂

## 🚀 Changes 

Updating PR template.

## ✅ Checklist

- [ ] All changes are covered by tests
- [ ] The changes are covered in docs
- [ ] The commits follow conventional commits standards

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- ☝️ make the title meaningful -->
+☝️ make the title meaningful
```
Comment (dozortsev) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: make it visible, so it will additionally remind people about this

**Comment by andreapiras** [comment]
Hey Can we also add cross browser check?

**Comment by dozortsev** [comment]
> Hey Can we also add cross browser check?

@andreapiras sure, what would you say and where exactly?

feel free to edit .md file via Github UI

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -9,11 +9,12 @@
...
 
 
 ## ✅ Checklist
 
 - [ ] All changes are covered by tests
+- [ ] All changes have been crossbrowser checked, especially IE11
 - [ ] The changes are covered in docs
-- [ ] The commits follow conventional commits standards
\ No newline at end of file
+- [ ] The commits follow conventional commits standards
```
Comment (twrdjpalmer) [inline .github/PULL_REQUEST_TEMPLATE.md:20]: Can we automate this check as opposed to having a checklist item?

Comment (dozortsev) [reply .github/PULL_REQUEST_TEMPLATE.md:20]: it's already automated as pre-commit check and as part of CI build

@andreapiras wdyt?

Comment (dozortsev) [reply .github/PULL_REQUEST_TEMPLATE.md:20]: I would probably tell people to give PR title name which follows CC standards as it's get pre-filled when you do "Squash & Merge"

Comment (dozortsev) [reply .github/PULL_REQUEST_TEMPLATE.md]: @twrdjpalmer done, wdyt?

https://github.com/transferwise/neptune-web/blob/improve-PR-template/.github/PULL_REQUEST_TEMPLATE.md

Comment (andreapiras) [reply .github/PULL_REQUEST_TEMPLATE.md]: yeah we have a commitlint job in CI, this is an old check

Comment (twrdjpalmer) [reply .github/PULL_REQUEST_TEMPLATE.md]: Ah I see. That is a fail—quite probable to get failures on merging to master. Bit like taking a sledgehammer to a nail, but worth considering disabling squash & merge, and leaving on the rebase option so we have to manage the commit messages before merging?

Comment (andreapiras) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yes it's not great , that's why we are merging PRs into master. We thought about rebase but that would pollute master with loads of commits. We need to take a decision on this.

Comment (twrdjpalmer) [reply .github/PULL_REQUEST_TEMPLATE.md]: Rebase & merge doesn't *have* to pollute master with loads of commits. It just puts the emphasis on the author to tailor their commits themselves. You can squash them all into one commit manually, or squash them into the smallest number of commits that make sense (some times having 2 or 3 commits in a PR makes more sense than a single behemoth of the commit)

**Pull request merged**